### DISCUSSION TITLE
Shapes Dotted lines or dashed lines etc

### DISCUSSION THREAD

**Discussion by xCheWy88** [flxzt/rnote#864] [Q&A]
I cannot seem to draw dashed or dotted shapes e.g. squares/rectangles. 

I did change the shape settings - Shaper > Shaper configuration > Fill style > dashed/dotted/ etc. Unless fill is for something else ? A bit confused, please help and thank you in advance

**Comment by silvasch** [comment]
There could be two reasons why this is:
1. Fill styles currently only matter when drawing a shape with the "Rough" shaper style
2. The fill color (to the right of the normal color) must not be transparent

**Reply by xCheWy88** [reply]
I see, so rnote cannot draw dashed line shapes, I just tried what you said, rough shaper style, and the dotts appear in the shape. I thought rnote could draw dashed or dotted etc outlines of a shape instead of a solid line. Thank you for your response though!
